test-support/src/main/java/org/springframework/ldap/test/EmbeddedLdapServer.java

```diff
@@ -85,7 +104,50 @@ public static EmbeddedLdapServer newEmbeddedServer(String defaultPartitionName,
     public void shutdown() throws Exception {
         ldapServer.stop();
         directoryService.shutdown();
-
-        FileUtils.deleteDirectory(workingDirectory);
     }
+
+	private static void initSchemaPartition(DefaultDirectoryService directoryService)
```

Comment:
Sorry...I think this is the last thing.

This is likely due to my own ignorance, but is there a reason we need to modify the code for this update? Is this required to update to apacheds 1.5.7 or is it a separate goal (in which case it might be best as a distinct issue)?